templates/base.html

```diff
@@ -107,17 +107,26 @@ <h1><a href="{{ SITEURL }}">{{ SITETITLE }}</a></h1>
       {% if pages or LINKS %}
       <nav>
         <ul class="list">
+          {# Open links in new window depending on the LINKS_IN_NEW_TAB setting #}
+          {% macro get_target(link) -%}
+            {% if LINKS_IN_NEW_TAB not in ('no', 'none', false, 0, 'external') or (LINKS_IN_NEW_TAB == "external" and not link.startswith("/") and not link.startswith(SITEURL)) %}
```

Comment:
based on my idea I believe this if is wrong.

We should only open in new ta if and only if LINKS_IN_NEW_TAB is True, None, or 'all'. Any other value should remain in the same tab.

'external' should only open if not relative or in the same domain, this part looks correct.

`LINKS_IN_NEW_TAB in (True, None, 'all')` should be the correct one, or similar.

Reply:
I believe in this case verbose makes more sense. We can have if all, elif external and else.

Reply:
Please note that `LINKS_IN_NEW_TAB is none` doesn't cover the case where `LINKS_IN_NEW_TAB` is undefined, which will be the case for existing installations. So in the end this boils down to whether you want to keep backwards compatibility or not. Let me know and I will write the verbose condition, hopefully it will be the last rewrite.

Reply:
I have finished this refactor, and included a fix to remove whitespace.